Add Office365/AzureAD Oauth (#202)
silinternational/wecarry-api: README.md

refer to domain/errorcodes.go.  In particular, the related codes are those 
that have a comment referring to actions.AuthRequest.

### Office365/AzureAD

> To enable authentication via AzureAD, an organization record will need to be created 

Reviewer: ```suggestion
To add an organization using AzureAD authentication, create a database organization record
```

Author: Sounds good.
